applications/app/services/TagPagePicker.scala

```diff
@@ -19,11 +19,11 @@ object TagPagePicker extends GuLogging {
       request.isJson,
       request.forceDCROff,
       request.forceDCR,
-      participatingInTest,
+      isSwitchedOn,
       dcrCouldRender(checks),
```

Comment:
We need to make sure that this these checks are sufficient – @guardian/dotcom-platform